lib/actions/enter.coffee

```diff
@@ -0,0 +1,75 @@
+###
+Copyright 2016 Resin.io
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+###
+
+getSubShellCommand = (command) ->
+	os = require('os')
+
+	# Assume Cygwin
+	if os.platform() is 'win32'
+		return {
+			program: 'sh'
+			args: [ '-c', command ]
+		}
+	else
+		return {
+			program: '/bin/sh'
+			args: [ '-c', command ]
+		}
+
+module.exports =
+	signature: 'enter <uuid>'
+	description: '(beta) get a shell into the running app container of a device'
+	help: '''
+		Use this command to get a shell into the running application container of
```

Comment:
Don't forget to generate documentation for this new command. You'll have to add this new action file to `capitanodoc.json` and run `npm run docs`.